starlark/value.go

```diff
@@ -1221,20 +1221,18 @@ func (s *Set) clone() *Set {
 	return set
 }
 
-func (s *Set) Union(iter Iterator) (Value, error) {
+func (s *Set) Union(other Iterator) (Value, error) {
 	set := s.clone()
-	var x Value
-	for iter.Next(&x) {
-		if err := set.Insert(x); err != nil {
-			return nil, err
-		}
+	if err := set.InsertAll(other); err != nil {
+		return nil, err
 	}
 	return set, nil
 }
 
-func (s *Set) InsertAll(iter Iterator) error {
+// TODO: rename to Update()
```

Comment:
We can't rename it, but you can add Update, make this function a wrapper around it, and add a `//go:fix inline` annotation on InsertAll, which will nudge people to use Update.

Reply:
Done.